apps/reference/docs/guides/api.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: api
-title: APIs
+title: Data APIs
 description: Auto-generating and Realtime APIs.
 sidebar_label: Overview
 ---
```

Comment:
In this file should we move the first sentence above the "Overview" heading and use the term "auto generating"?

Eg:

"Supabase auto-generates three types of API directly from your database schema."

## Overview

- REST ....

Reply:
Done—took out the "Overview" heading as it's unnecessary.